src/services/ColorThemePlugin.js

```diff
@@ -12,18 +12,43 @@ const getDefaultOptions = () => ({
 })
 
 export const ColorThemePlugin = {
-  install (Vue, options) {
+  install (Vue, options) { // options = [ { name: 'myName', themes: { primary: '', ... }} ]
```

Comment:
Multiple themes are never needed. Should be one object.

Reply:
In vuestic-admin used multiple themes (original/corporate)
These changes need for possibility change the corporate theme by user